packages/components/src/tools-panel/tools-panel-item/hook.ts

```diff
@@ -28,14 +31,14 @@ export function useToolsPanelItem( props ) {
 	const cx = useCx();
 	const classes = useMemo( () => {
 		return cx( styles.ToolsPanelItem, className );
-	} );
+	}, [ styles.ToolsPanelItem, className ] );
```

Comment:
I don't believe that `styles.ToolsPanelItem` is required here, being an import

```suggestion
	}, [ className ] );
```